test/element-visibility.test.js

```diff
@@ -94,7 +95,7 @@ describe('o-element-visibility', function() {
 			oElemVis.track(outview);
 		});
 
-		it('should recalculate the position of tracked elements once you scroll if height of body changed', function(done) {
+		it.skip('should recalculate the position of tracked elements once you scroll if height of body changed', function(done) {
```

Comment:
Um, skip?

Reply:
This test never ran before because of the `isPhantom` call, I think we can look into fixing and running this test in a subsequent PR.

Reply:
Can we have an issue for this please?

Reply:
Issue added https://github.com/Financial-Times/o-element-visibility/issues/21